concepts/numbers/about.md

```diff
@@ -1,8 +1,50 @@
 # About
 
-## Type Conversion
+Go contains basic numeric types that can represent sets of either integer or
+floating-point values. There a different types depending on the size of value
+you require and the architecture of the computer where the application is
+running (e.g. 32-bit and 64-bit).
+
+For the sake of this exercise you will only be dealing with:
```

Comment:
These concepts are not necessarily viewed within the context of a specific exercise, so please remove any mention of a particular exercise.